acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/PantheonNode.java

```diff
@@ -255,6 +261,22 @@ public void useWebSocketsForJsonRpc() {
     useWsForJsonRpc = true;
   }
 
+  /** All future JSON-RPC calls will include the authentication token. */
+  @Override
+  public void useAuthenticationTokenInHeaderForJsonRpc(final String token) {
+
+    if (jsonRequestFactories != null) {
+      jsonRequestFactories.shutdown();
```

Comment:
need to add `jsonRequestFactoies = null;` here, it's also wrong in useWebSocketsForJsonRpc but we haven't managed to race condition it yet.

Reply:
👍

Reply:
This caused one AT to fail so putting it back in
java.lang.RuntimeException: org.web3j.protocol.exceptions.ClientConnectionException: Invalid response received: 400; Websocket endpoint can't handle HTTP requests

Reply:
RpcApisTogglesAcceptanceTest.shouldSucceedConnectingToNodeWithWsRpcEnabled